internal/commands/inspect_builder_test.go

```diff
@@ -60,14 +60,12 @@ func testInspectBuilderCommand(t *testing.T, when spec.G, it spec.S) {
 
 	when("#Get", func() {
 		when("image cannot be found", func() {
-			it("logs 'Not present'", func() {
+			it("logs 'ERROR Unable'", func() {
```

Comment:
This should read `it("errors when no image is found")`.

Reply:
We also need at least one test (preferably two) where we test when only one of the two is returned and ensure it doesn't error.

Reply:
Done and done after reorganizing test data.